CODE/Logicytics.py

```
@@ -272,12 +274,12 @@
 
         log.debug("Using threading")
         threads = []
-        execution_list = generate_execution_list(action)
-        for index, file in enumerate(execution_list):
+        EXECUTION_LIST = generate_execution_list()
+        for index, file in enumerate(EXECUTION_LIST):
```

Ruff (0.8.0)

Loop control variable `file` not used within loop body

(B007)